### ISSUE TITLE
About the commands to pretraining the model

### ISSUE THREAD

**Issue by JiaxinZhuang** [Ziyan-Huang/STU-Net#11]
Dear author, thanks for your impressive work.
Could you please offer some commands for how to pertain the mode on the totalsegmentator dataset?

**Comment by Ziyan-Huang** [comment]
Hello
You can use the standard nnUNet training script `nnUNet_train `to train the Trainer provided in our repository (STUNetTrainer) for 4000 epochs on the TotalSegmentator dataset.
Here is an example
`nnUNet_train STUNetTrainer 3d_fullres TASKID FOLD`
Please replace TASKID with the task ID for the TotalSegmentator dataset and FOLD with the fold number you want to train on.

**Comment by Ziyan-Huang** [comment]
Hello，@Airliin 

Thank you for your question! In the context of multi-class segmentation in medical imaging, especially when using Totalsegmentator which labels across 104 classes, it's common for some cases not to contain all existing labels. This is normal and expected as each image or case may only present a subset of conditions or structures.

If a category does not exist in a particular sample, it is typical not to mark it specially. Instead, you would just label the categories that do exist in each sample. The model will learn to recognize and segment different categories from the samples where those categories are present.

**Comment by Airliin** [comment]
Thank you for the prompt response!
